src/utils/RootTypeInitializer.ts

```diff
@@ -0,0 +1,60 @@
+import ResourceType from '@/models/ResourceType';
+import rootTypes from '@/utils/rootTypes';
+import Winston from 'winston';
+
+export default class RootTypeInitializer {
+  // region public static methods
+  // endregion
+
+  // region private static methods
+  // endregion
+
+  // region public members
+  // endregion
+
+  // region private members
+  // endregion
+
+  // region constructor
+  // endregion
+
+  // region public methods
+  public static async initializeRootTypes(): Promise<void> {
+    const resourceTypeCount = await ResourceType.find().estimatedDocumentCount();
+
+    if (resourceTypeCount !== 0) {
+      return;
+    }
+
+    Winston.info('Begin initializing root types...');
+
+    for (const rootType of rootTypes) {
+      const resourceRootType = new ResourceType(rootType);
+
+      if (rootType.parentType) {
+        const parentResourceType = await ResourceType.findOne({ name: rootType.parentType });
```

Comment:
Consider extracting line 35 - 43 to an own method inside the ResourceType class. Something like 
```ts
setParentResourceType(resourceTypeName: string) {...}
```